demisto_sdk/commands/common/hook_validations/id.py

```diff
@@ -41,41 +40,22 @@ class IDSetValidator(BaseValidator):
     MAPPERS_SECTION = "Mappers"
     INCIDENT_TYPES_SECTION = "IncidentTypes"
 
-    ID_SET_PATH = "./Tests/id_set.json"
-
     def __init__(self, is_test_run=False, is_circle=False, configuration=Configuration(), ignored_errors=None,
-                 print_as_warnings=False, suppress_print=False, id_set_path: str = None):
+                 print_as_warnings=False, suppress_print=False, id_set_file=None):
         super().__init__(ignored_errors=ignored_errors, print_as_warnings=print_as_warnings,
                          suppress_print=suppress_print)
         self.is_circle = is_circle
         self.configuration = configuration
         if not is_test_run and self.is_circle:
-            self.id_set_path = id_set_path or self.ID_SET_PATH
-            self.id_set = self.load_id_set()
-            self.script_set = self.id_set[self.SCRIPTS_SECTION]
-            self.playbook_set = self.id_set[self.PLAYBOOK_SECTION]
-            self.integration_set = self.id_set[self.INTEGRATION_SECTION]
-            self.test_playbook_set = self.id_set[self.TEST_PLAYBOOK_SECTION]
-            self.classifiers_set = self.id_set[self.CLASSIFIERS_SECTION]
-            self.layouts_set = self.id_set[self.LAYOUTS_SECTION]
-            self.mappers_set = self.id_set[self.MAPPERS_SECTION]
-            self.incident_types_set = self.id_set[self.INCIDENT_TYPES_SECTION]
-
-    def load_id_set(self):
-        with open(self.id_set_path, 'r') as id_set_file:
-            try:
-                id_set = json.load(id_set_file)
-            except ValueError as ex:
-                if "Expecting property name" in str(ex):
-                    error_message, error_code = Errors.id_set_conflicts()
-                    if self.handle_error(error_message, error_code, file_path="id_set.json"):
-                        raise
-                    else:
-                        pass
-
-                raise
-
-            return id_set
+            self.id_set_file = id_set_file
```

Comment:
you changed the name here from `id_set` to `id_set_file` - yet you did not change this anywhere else so is this redundant?

Reply:
I changed it from `id_set_path` to `id_set_file`. which I have changed all its occurrences.
Or maybe I'm misunderstanding you?